src/ui/server/controllers/insurance/your-buyer/validation/rules/address.test.ts

```diff
@@ -0,0 +1,51 @@
+import addressRules, { MAXIMUM } from './address';
+import { FIELD_IDS } from '../../../../../constants';
+import { ERROR_MESSAGES } from '../../../../../content-strings';
+import generateValidationErrors from '../../../../../helpers/validation';
+
+const {
+  YOUR_BUYER: {
+    COMPANY_OR_ORGANISATION: { ADDRESS: FIELD_ID },
+  },
+} = FIELD_IDS.INSURANCE;
+
+const {
+  INSURANCE: {
+    YOUR_BUYER: {
+      COMPANY_OR_ORGANISATION: { [FIELD_ID]: ERROR_MESSAGE },
```

Comment:
is there a better way of this? it looks a bit confusing

Reply:
I can't think of another way to do it without breaking eslint rules - in this file we get the field ID, and get the error message for that field ID - they have the same name/ID in both files. Which is good IMO. We have this approach in other places as well

So we get the `FIELD_ID` const from field-ids, then we get the error message dynamically with the `FIELD_ID` constant but label it `ERROR_MESSAGE`. Makes the actual tests are easier to read because then we can do e.g `ERROR_MESSAGE.IS_EMPTY`.

Do you have another idea or something we could do to make it clearer?

Reply:
no this seems fine then!